scripts/run.py

```diff
@@ -156,24 +151,16 @@ def main(computation, backend, path, url, model_point, number, mode):
     https://github.com/pyhf/pyhf-benchmark
 
     """
-    system = SystemStats()
-    system.start()
-
-    if computation == "mle":
-        if backend == "tensorflow":
-            tf.get_logger().setLevel("ERROR")
 
-        pyhf.set_backend(backend)
-        print(f"Backend set to: {backend}")
+    if backend.find("[") != -1:
```

Comment:
Is this just catching if a list of options is passed in? Just curious.

Reply:
As we discussed in [issue#21](https://github.com/pyhf/pyhf-benchmark/issues/21), this code aims to check if the user makes use of one backend or several backends in one run. If the user just uses one backend, he does not need to add brackets in command line.